### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by BenLeggiero** [bigfive/atom-sublime-select#137]
1. hold <kbd>alt</kbd>
2. Click and drag

This does _not_ occur when Sublime-Style-Column-Selection is disabled

**Atom Version**: 1.13.0
**Electron Version**: 1.3.13
**System**: Microsoft Windows 10 Pro
**Thrown From**: [bracket-matcher](https://github.com/atom/bracket-matcher) package, v0.85.1

### Stack Trace
See complete **3MB** stack trace uploaded to Google Drive: https://drive.google.com/open?id=0BzaVNSW3H07tY1dXOTVORDhJbmc

### Commands

```
     -1:48.8.0 core:move-left (input.hidden-input)
     -1:48.4.0 editor:newline (input.hidden-input)
     -1:43.1.0 core:backspace (input.hidden-input)
     -1:41.8.0 editor:newline (input.hidden-input)
     -1:40.4.0 core:undo (input.hidden-input)
     -1:35.7.0 core:move-left (input.hidden-input)
     -1:35.2.0 editor:newline (input.hidden-input)
     -1:33.6.0 core:backspace (input.hidden-input)
     -1:27.2.0 core:save (input.hidden-input)
     -0:37.7.0 snippets:next-tab-stop (input.hidden-input)
     -0:37.7.0 snippets:expand (input.hidden-input)
     -0:37.7.0 editor:indent (input.hidden-input)
  3x -0:31.3.0 core:backspace (input.hidden-input)
     -0:26.1.0 core:cut (input.hidden-input)
  2x -0:26 core:undo (input.hidden-input)
  3x -0:22.8.0 core:backspace (input.hidden-input)
```

### Config

```json
{
  "core": {
    "telemetryConsent": "limited",
    "themes": [
      "atom-material-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-material-syntax, v1.0.2 (inactive)
atom-material-ui, v1.3.9 (active)
autocomplete-swift, v1.0.3 (active)
language-swift, v0.5.0 (active)
Remote-FTP, v0.9.4 (active)
simple-drag-drop-text, v0.3.1 (active)
Sublime-Style-Column-Selection, v1.7.2 (active)
atom-dark-syntax, v0.28.0 (active)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (inactive)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (inactive)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.14.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.33.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.43.2 (active)
bracket-matcher, v0.85.1 (active)
command-palette, v0.39.1 (inactive)
deprecation-cop, v0.55.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.2.0 (active)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.1 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.159.1 (active)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.5 (active)
status-bar, v1.6.0 (active)
styleguide, v0.48.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (active)
timecop, v0.33.2 (active)
tree-view, v0.211.1 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.39.0 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.1 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by BenLeggiero** [comment]
This seems to be linked with trying to select a column on an empty line; it still works fine for me when I don't do that

**Comment by d4rkd0s** [comment]
![image](https://cloud.githubusercontent.com/assets/2348814/22280982/ce781ee0-e299-11e6-8dda-7ce9f942c5be.png)

**Comment by d4rkd0s** [comment]
Only plugin installed was Sublime Column Style select. This started happening to me when I updated to Atom 1.13.0

**Comment by bigfive** [comment]
I've just merged a fix for issues with 1.13 can you please update and recheck your issue

**Comment by d4rkd0s** [comment]
Looks like its working perfectly after updating! Tank you @bigfive 👍

**Comment by d4rkd0s** [comment]
@BenLeggiero can you test after an update as well?

**Comment by d4rkd0s** [comment]
Thx @bigfive  👍

**Comment by BenLeggiero** [comment]
This seems to have been fixed in Sublime-Style-Column-Selection 1.7.3 on Atom 1.14.2
